Library/Formula/showkey.rb

```diff
@@ -0,0 +1,62 @@
+class Showkey < Formula
+  desc "Command-line keystroke visualizer"
+  homepage "http://www.catb.org/~esr/showkey"
+  url "http://www.catb.org/~esr/showkey/showkey-1.7.tar.gz"
+  sha256 "0383a508c29df9a80b294a805a38f38d2dee6d2687e43c5ece3c5901220fb44d"
+
+  head do
+    url "git://thyrsus.com/repositories/showkey.git"
+    depends_on "xmlto" => :build
+    # Disable checking XML validity by xmlto, as xslproc is often configured to
+    # avoid downloading DTDs. This can result in a xml to man conversion error.
+    patch :DATA
+  end
+
+  def install
+    # There is no configure script, and makefile has the wrong prefixes
```

Comment:
It’s usually better to use `inreplace` to fix the `Makefile` then use it so that the issue is clear when reading the code. If we install things manually someone reading the code can’t know if it’s because the `Makefile` is broken or because it doesn’t have an e.g. `install` target.